markets/spot-market/contracts/interfaces/ISpotMarketFactoryModule.sol

```diff
@@ -15,6 +15,11 @@ interface ISpotMarketFactoryModule is IMarket {
      */
     error NotNominated(address addr);
 
+    /**
+     * @notice Thrown when createSynth is called with zero-address synth owner
+     */
+    error InvalidMarketOwner();
```

Comment:
you know we also have `ParameterError.InvalidParameter("owner", "cannot be 0")` for this as well, rigth?

Reply:
wow this is very useful.  i did not know that;  i'll make a note to come back and fix some of these.  could definitely reuse this